test/index.js

```diff
@@ -0,0 +1,11 @@
+// Directions: https://github.com/webpack-contrib/istanbul-instrumenter-loader
```

Comment:
This is how it is recommended on the webpack plugin.  https://github.com/webpack-contrib/istanbul-instrumenter-loader.  I think it needs to be this way to work with karma as well.  test/index.js is the entry point in karma and then it searches through the src/ directory for index.js and requires them.  Pretty sure I tried to avoid this and couldn't get it to work without implementing that structure.